src/clone.d

```diff
@@ -920,7 +920,7 @@ extern (C++) FuncDeclaration buildPostBlit(StructDeclaration sd, Scope* sc)
 
     /* Build our own "postblit" which executes a
      */
-    if (a || (stc & STCdisable))
+    if ((a && a.dim) || (stc & STCdisable))
```

Comment:
Would be nicer with a comment, or even better a cleaner control flow. From the code above it takes a while to see how this can be empty.